examples/kanidm

```diff
@@ -0,0 +1,46 @@
+# This should be at /etc/kanidm/config or ~/.config/kanidm,
```

Comment:
teh kanidm file will include radius config now? But the rust tools won't process that? Actually I think they'll ignore it, but seems odd .... I'm sure you have a reason for this :)

Reply:
Yeah, I figured the radius server is a "client" so it's just easier to have one place/format to look at.

And yep, `kanidm` will happily ignore other things.